### DISCUSSION TITLE
How do I remove an item from a playlist

### DISCUSSION THREAD

**Discussion by FFFluoride** [sayaka-sh/spmp#411] [Q&A]
I want to remove an item but don't know how

**Comment by FFFluoride** [comment]
I'd really appreciate an answer. No worries if you're busy though. Just thought I should leave a comment in case my question was missed.

**Comment by sayaka-sh** [comment]
Sorry, removing items from playlists is broken for both local and remote playlists at the moment. Normally, you would just long-press or multi-select items in the playlist to remove them, but currently the option either doesn't appear or throws an exception.

**Reply by FFFluoride** [reply]
Thanks for letting me know
